healthcheck/healthcheck.go

```diff
@@ -12,16 +12,16 @@ const language = "go"
 
 // HealthCheck represents the app's health check, including its component checks
 type HealthCheck struct {
-	Status                   string        `json:"status"`
-	Version                  VersionInfo   `json:"version"`
-	Uptime                   time.Duration `json:"uptime"`
-	StartTime                time.Time     `json:"start_time"`
-	Checks                   []*Check      `json:"checks"`
-	Started                  bool          `json:"-"`
-	Interval                 time.Duration `json:"-"`
-	CriticalErrorTimeout     time.Duration `json:"-"`
-	TimeOfFirstCriticalError time.Time     `json:"-"`
-	Tickers                  []*ticker     `json:"-"`
+	Status                   string          `json:"status"`
+	Version                  VersionInfo     `json:"version"`
+	Uptime                   time.Duration   `json:"uptime"`
+	StartTime                time.Time       `json:"start_time"`
+	Checks                   []*Check        `json:"checks"`
+	interval                 time.Duration   `json:"-"`
```

Comment:
Unexported fields do not need a json tag of `-` as they will not be encoded or decoded when using the json.Marshal or json.Unmarshal